src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+
+  margin: auto 60px;
+}
+
+.li-nav {
+  list-style-type: none;
+  font-family: "Roboto", sans-serif;
```

Comment:
we can add our font in element like `body`, `html` or `*` then it will be applied to every element on the page